site/ja/blog/new-in-devtools-93/index.md

```diff
@@ -0,0 +1,248 @@
+-->
+Manifestペインで[new note URL](https://wicg.github.io/manifest-incubations/index.html#dfn-note_taking)が表示されるようになります。
+
+<!--
+Currently on Chrome OS (CrOS), Chrome Apps and Android Apps that declare a "new-note" capability may be selected as a note-taking app in the Stylus settings (shows up if the CrOS device has been used with a stylus). When selected as a note-taking app, the app can be launched from the stylus palette's "Create Note" button. Adding `new-note-url` field in the application manifest is part of the effort to add equivalent functionality to web apps.
+-->
+現在Chrome OS（CrOS）では、"new-note" capabilityを宣言したChrome Apps、Android Appsは（CrOSデバイスがスタイラスを使用していたら現れる）Stylus settingsの中でnote-taking appとして選択できるようになります。note-taking appとして選択すると、そのアプリはスタイラスパレットの「Create Note」ボタンで起動できるようになります。アプリケーションのマニフェストに`new-note-url`フィールドを追加することはウェブアプリに同様の機能を追加するための作業のひとつです。
+
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/2Cwggroar7pNesfAQi4K.png", alt="New note URL in the Manifest pane", width="800", height="477" %}
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/51f8aaf568db256f3390c37393d294c82017565e #}
+
+Chromium issue: [1185678](https://crbug.com/1185678)
+
+
+<!--
+## Fixed CSS matching selectors {: #matching-selectors }
+-->
+## CSSにマッチするセレクタ表示の修正 {: #matching-selectors }
+<!--
+DevTools fixed the CSS matching selectors, it was not working in the last release.
+-->
+DevToolsはCSSにマッチしたセレクタの表示を修正しました。これは前回のリリースでは動作していませんでした。
+
+<!--
+The comma separated selectors in the **Styles** pane are colored differently depending on whether they match the selected DOM node:
+-->
+**Styles**パネルのカンマで分割されたセレクタは、選択されたDOMノードにマッチするかどうかに応じて異なる色が使用されます。
+
+<!--
+- An unmatched portion is shown in a light grey.
+- A matching selector portion is shown in black.
+-->
+- マッチしない部分は明るい灰色で表示される
+- マッチするセレクタの部分は黒で表示される
+
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/O7CoHBrKA9cVKci1SM0M.png", alt="CSS matching selectors", width="800", height="477" %}
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/123eac3c8ceeb2e788aa4756d3104db0265f9ad3 #}
+
+Chromium issue: [1219153](https://crbug.com/1219153)
+
+
+<!--
+## Pretty-printing JSON responses in the Network panel {: #pretty-print-json }
+-->
+## NetworkパネルでのJSONレスポンスのプリティプリント {: #pretty-print-json }
+<!--
+You can now pretty print JSON responses in the **Network** panel.
+-->
+NetworkパネルでJSONレスポンスがプリティプリントされようになりました。
+
+
+<!--
+Open a JSON response in the **Network** panel, click on the `{}` icon to pretty-print it.
+-->
+**Network**パネルでJSONレスポンスを開き、`{}`アイコンをクリックするとプリティプリントされます。
+
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/x2NKXwJPzjycjeD7cLH6.png", alt=" Pretty-printing JSON responses in the Network panel", width="800", height="523" %}
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/250c88b4d02da283cd0a96204b1592f59fda2fcb #}
+
+Chromium bug: [998674](https://crbug.com/998674) 
+
+{% include 'partials/devtools/ja/reach-out.md' %}
```

Comment:
Please help to translate this file content into Japanese.

Reply:
done